### ISSUE TITLE
_imagingft.cpython-39-x86_64-linux-gnu.so is making a Segment Fault inside OBS Studio

### ISSUE THREAD

**Issue by MarioMey** [python-pillow/Pillow#5693]
I just migrated from Debian 10 to 11 and I'm doing every software works in this new setup. I use OBS Studio with a script of mine to make lot of things. In one of them, I use `python-imagesearch` (that uses Pillow) to detect something in the screen (some buttons from Zoom Meeting sofware). In Debian 10, it worked OK. Now, I get a Segment Fault. I arrived to this file `_imagingft.cpython-39-x86_64-linux-gnu.so` that it is doing the Segment Fault.

I would like to compile Pillow to see if the problem is in repository (I installed using `python3.9 -m pip install Pillow`)... but I don't know how to do it. There is no documentation.

### What are your OS, Python and Pillow versions?
Debian 11, Python 3.9.2, Pillow 8.3.1

**Comment by radarhere** [comment]
Hi. To compile Pillow from source, you can either ask pip to compile it from source - `python3.9 -m pip install Pillow --no-binary :all:` - or you can download the Pillow source code, move into the directory, and `python3.9 setup.py install`.

There are various dependencies that you may find that you need.

`sudo apt-get install python3-dev python3-setuptools libfreetype6-dev libfribidi-dev libharfbuzz-dev libjpeg-turbo-progs libjpeg62-turbo-dev liblcms2-dev libopenjp2-7-dev libtiff5-dev libwebp-dev zlib1g-dev`

**Comment by radarhere** [comment]
Would we be able to get a complete log from the segfault? Or are you only after advice on how to compile Pillow?

**Comment by nulano** [comment]
This sounds to me like a duplicate of #5637.

Does the compiled binary wheel from https://github.com/python-pillow/Pillow/issues/5637#issuecomment-891379153 help?

**Comment by MarioMey** [comment]
I installed binary wheel and it worked! It was not necessary to compile it. Thanks to both.
